egs/csj/s5/RESULTS

```diff
@@ -1,134 +1,212 @@
-%WER 13.35 [ 3476 / 26028, 387 ins, 789 del, 2300 sub ] exp/tri4_fmmi_b0.1/decode_eval1_it8_csj/wer_13_0.5
-%WER 14.01 [ 3646 / 26028, 396 ins, 826 del, 2424 sub ] exp/tri4_mmi_b0.1/decode_eval1_1.mdl_csj/wer_17_0.0
-%WER 13.70 [ 3567 / 26028, 315 ins, 896 del, 2356 sub ] exp/tri4_mmi_b0.1/decode_eval1_2.mdl_csj/wer_15_0.5
-%WER 13.60 [ 3540 / 26028, 331 ins, 878 del, 2331 sub ] exp/tri4_mmi_b0.1/decode_eval1_3.mdl_csj/wer_15_0.5
-%WER 13.55 [ 3526 / 26028, 355 ins, 837 del, 2334 sub ] exp/tri4_mmi_b0.1/decode_eval1_4.mdl_csj/wer_14_0.5
-%WER 11.71 [ 3047 / 26028, 288 ins, 771 del, 1988 sub ] exp/dnn5b_pretrain-dbn_dnn/decode_eval1_csj/wer_15_0.0
-%WER 11.07 [ 2880 / 26028, 340 ins, 619 del, 1921 sub ] exp/dnn5b_pretrain-dbn_dnn_smbr/decode_eval1_csj/wer_12_0.5
-%WER 11.03 [ 2871 / 26028, 335 ins, 638 del, 1898 sub ] exp/dnn5b_pretrain-dbn_dnn_smbr_i1lats/decode_eval1_csj/wer_14_0.5
+%WER 22.32 [ 5809 / 26028, 445 ins, 1539 del, 3825 sub ] exp/tri1/decode_eval1_csj/wer_13_0.0
+%WER 21.05 [ 5480 / 26028, 436 ins, 1419 del, 3625 sub ] exp/tri2/decode_eval1_csj/wer_13_0.0
+%WER 16.86 [ 4388 / 26028, 452 ins, 1036 del, 2900 sub ] exp/tri3/decode_eval1_csj/wer_17_0.0
+%WER 18.11 [ 4714 / 26028, 394 ins, 1212 del, 3108 sub ] exp/tri3/decode_eval1_csj_nosp/wer_17_0.0
+%WER 14.73 [ 3834 / 26028, 419 ins, 887 del, 2528 sub ] exp/tri4/decode_eval1_csj/wer_17_0.0
+%WER 17.07 [ 4443 / 26028, 504 ins, 947 del, 2992 sub ] exp/tri4/decode_eval1_csj.si/wer_17_0.0
+%WER 14.09 [ 3668 / 26028, 416 ins, 816 del, 2436 sub ] exp/tri4_fmmi_b0.1/decode_eval1_it4_csj/wer_16_0.0
+%WER 13.57 [ 3533 / 26028, 320 ins, 859 del, 2354 sub ] exp/tri4_fmmi_b0.1/decode_eval1_it5_csj/wer_14_0.5
+%WER 13.57 [ 3531 / 26028, 354 ins, 836 del, 2341 sub ] exp/tri4_fmmi_b0.1/decode_eval1_it6_csj/wer_13_0.5
+%WER 13.44 [ 3499 / 26028, 369 ins, 822 del, 2308 sub ] exp/tri4_fmmi_b0.1/decode_eval1_it7_csj/wer_13_0.5
+%WER 13.44 [ 3498 / 26028, 385 ins, 807 del, 2306 sub ] exp/tri4_fmmi_b0.1/decode_eval1_it8_csj/wer_13_0.5
+%WER 13.92 [ 3623 / 26028, 396 ins, 834 del, 2393 sub ] exp/tri4_mmi_b0.1/decode_eval1_1.mdl_csj/wer_16_0.0
+%WER 13.73 [ 3574 / 26028, 303 ins, 916 del, 2355 sub ] exp/tri4_mmi_b0.1/decode_eval1_2.mdl_csj/wer_15_0.5
+%WER 13.68 [ 3561 / 26028, 335 ins, 900 del, 2326 sub ] exp/tri4_mmi_b0.1/decode_eval1_3.mdl_csj/wer_15_0.5
+%WER 13.65 [ 3554 / 26028, 336 ins, 884 del, 2334 sub ] exp/tri4_mmi_b0.1/decode_eval1_4.mdl_csj/wer_15_0.5
+%WER 11.69 [ 3042 / 26028, 325 ins, 711 del, 2006 sub ] exp/dnn5b_pretrain-dbn_dnn/decode_eval1_csj/wer_13_0.0
+%WER 11.10 [ 2890 / 26028, 300 ins, 678 del, 1912 sub ] exp/dnn5b_pretrain-dbn_dnn_smbr/decode_eval1_csj/wer_15_0.5
+%WER 10.96 [ 2853 / 26028, 319 ins, 642 del, 1892 sub ] exp/dnn5b_pretrain-dbn_dnn_smbr_i1lats/decode_eval1_csj/wer_15_0.5
+%WER 11.24 [ 2926 / 26028, 263 ins, 658 del, 2005 sub ] exp/chain/tdnn_csj/decode_eval1/wer_12_0.5
```

Comment:
Are these results in the RESULTS file up to date?

Reply:
No. As the RESULT file contains 3 different data prepare modes(0,2,3), I just have the result of default mode(0), and the experiments of modes(2,3) are running. So I prepare to update the "RESULT" file after all the experiments finish.

Reply:
Otherwise, Need I rerun the local/nnet/run_dnn.sh and local/nnet3/run_tdnn.sh?
Now I have the result of gmm-hmm(tri1,tri2, tri3, tri4, tri4_fmmi, tri4_mmi) model and chain/tdnn model. 
I don't have the new result to update the result of nnet1/dnn(e.g. exp/dnn5b_pretrain-dbn_dnn_smbr, I'm afraid it wll be slow.), and the result of nnet3/tdnn(e.g. exp/nnet3/nnet_tdnn. A problem is recipe looks old, it still use make_configs.py to generate the topo. Need I rewrite it?)